activities/codebase-stewardship-assessment/index.md

```diff
@@ -0,0 +1,17 @@
+---
+type: resource
+---
+
+# Codebase stewarship assessment
+
+In assessment we evaluate whether a codebase can be [incubated in codebase stewardship](../codebase-stewardship/lifecycle.md).
```

Comment:
I believe this sentence is incomplete, it feels like it is missing a "or not" in the end (but I am not a native English speaker, so I might be wrong here).

Reply:
I think this is normal in English (and Dutch). 

As this pull request only moves this file I think we should solve this in a separate pull request if this is an issue.

Reply:
This is grammatically correct in English.